ambry-clustermap/src/test/java/com.github.ambry.clustermap/CloudReplicaTest.java

```diff
@@ -57,7 +55,7 @@ public void basicTest() {
     CloudReplica cloudReplica = new CloudReplica(new MockPartitionId(), cloudDataNode);
     assertEquals("Wrong mount path", mockPartitionId.toPathString(), cloudReplica.getMountPath());
     assertEquals("Wrong replica path",
-        GetRequest.Cloud_Replica_Keyword + File.separator + mockPartitionId.toPathString() + File.separator
+        CloudReplica.Cloud_Replica_Keyword + File.separator + mockPartitionId.toPathString() + File.separator
```

Comment:
Remove the constant from GetRequest too.

Reply:
will do.